Dockerfile

```diff
@@ -1,19 +1,20 @@
 FROM {{%DRYDOCK_ORG%}}/microbase:{{%TAG%}}
 
-ENV REQPROC_PATH /home/shippable/reqProc
-ADD . $REQPROC_PATH
-RUN cd $REQPROC_PATH && npm install
+# Prefix all image ENVs with IMAGE_ so there are no confusions.
+ENV IMAGE_REQPROC_DIR /home/shippable/reqProc
```

Comment:
we should get rid of /home/shipapble here. it should be /root 
- all new workflow artifacts should be stored in /root